src/components/EmptyState/EmptyState.scss

```diff
@@ -142,6 +142,7 @@ $centered-illustration-width: rem(226px);
 
 .centeredLayout {
   .Section {
+    left: 0;
```

Comment:
This looks out of place 🤔

Reply:
I missed this override in my centeredLayout changes. Not overriding the left property was causing the centeredLayout to be pushed to the right by 2rems or something like that. Ashwin is fixing it for me in this PR